tests/adapters_t/batchsystems_t/test_htcondor.py

```diff
@@ -156,7 +158,23 @@ def test_get_resource_ratios(self):
             run_async(
                 self.htcondor_adapter.get_resource_ratios, drone_uuid="not_exists"
             ),
-            {},
+            [],
+        )
+        self.mock_async_run_command.assert_called_with(self.command)
```

Comment:
```suggestion
        self.mock_async_run_command.assert_called_with(self.command)
        self.mock_async_run_command.reset_mock()
```

Reply:
Thanks, good to know ;) Next time I will remember.